drivers/sht1x/Kconfig

```diff
@@ -5,32 +5,53 @@
 # directory for more details.
 #
 
-if TEST_KCONFIG
+menuconfig MODULE_SHT1X
+    bool
+    prompt "SHT10/SHT11/SHT15 Humidity and Temperature Sensor"
+    depends on HAS_PERIPH_GPIO
+    depends on TEST_KCONFIG
+    select MODULE_PERIPH_GPIO
+    select MODULE_XTIMER
+
+if MODULE_SHT1X
 
 choice
     bool "SHT10/SHT11/SHT15 Humidity and Temperature Sensor"
-    optional
-    depends on HAS_PERIPH_GPIO
+    default MODULE_SHT10 if HAVE_SHT10
```

Comment:
Maybe change the prompt on the choice? Something like `Sensor variant` or `Sensor model`

Reply:
Done